src/Resources/skeleton/command/Command.tpl.php

```diff
@@ -9,11 +9,19 @@
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Console\Style\SymfonyStyle;
 
+<?php if ($use_attributes): ?>
```

Comment:
It's more complex than this. `$use_attributes` tells us if the PHP version supports attributes. But in this case, we *also* need to see if the version of Symfony has the `ConsoleCommand` class. Also, this will need a `use` statement on top.

I know we're in the process of fixing some test failures to due upstream changes (so it's hard to look at CI), but let's make sure that if we misconfigure this somehow (e.g. forget the `use` statement) that it *does* cause a test to fail.